apps/sms/views/conversation/js/conversation.js

```diff
@@ -2915,9 +2923,9 @@ var ConversationView = {
     }
 
     // Update thread timestamp. Will be removed in bug 958105.
-    var thread = Threads.active;
-    if (thread) {
-      thread.timestamp = Date.now();
+    if (this.currentThread) {
+      // TODO: Will need to replace with Conversation service method.
```

Comment:
Ok, so we can just remove this part right in this patch, or leave this for bug 958105? I'm fine to address this since the change should be relatively small.